styles.css

```diff
@@ -191,3 +191,11 @@ body {
   border-radius: 15px;
   background: skyblue;
 }
+
+@media screen and (max-width: 800px){
+  body {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
```

Comment:
Would be nice if the country selector displays horizontally here. Don't you think?